mmaction/datasets/rawframe_dataset.py

```diff
@@ -52,6 +52,8 @@ class RawframeDataset(BaseDataset):
             Default: False.
         filename_tmpl (str): Template for each filename.
             Default: 'img_{:05}.jpg'.
+        with_offset (bool): Determines whether the offset information is in
+            ann_file. If not, frame_idx starts from 1. Default: False.
```

Comment:
Need to give example in docstring for untrimmed datasets annotation like #line22 - #line43